cdap-common/src/main/java/io/cdap/cdap/common/guice/preview/PreviewDiscoveryRuntimeModule.java

```diff
@@ -38,8 +44,11 @@ protected void configure() {
     // and router can route directly to preview service. for other services we want to use local discovery service.
     bind(DiscoveryService.class).annotatedWith(
       Names.named("shared-discovery-service")).toInstance(discoveryService);
```

Comment:
Is this used? I tried looking for usage of `shared-discovery-service` and only find it using in this module.

Reply:
Don't think it is used, will remove it